src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.fileReader;
```

Comment:
The class name `fileReader` does not follow Java naming conventions. Class names should be in PascalCase, such as `FileReader`.